Sources/SPMTestSupport/misc.swift

```diff
@@ -364,3 +364,20 @@ extension RelativePath: ExpressibleByStringInterpolation {
         try! self.init(validating: value)
     }
 }
+
+extension InitPackage {
+    public convenience init(
+        name: String,
+        packageType: PackageType,
+        destinationPath: AbsolutePath,
+        fileSystem: FileSystem
+    ) throws {
+        try self.init(
+            name: name,
+            options: InitPackageOptions(packageType: packageType),
+            destinationPath: destinationPath,
+            installedSwiftPMConfiguration: .default,
+            fileSystem: fileSystem
+        )
+    }
+}
```

Comment:
What is this needed for?

Reply:
This was my solution to not have to update all the tests :)